crates/ra_assists/src/doc_tests.rs

```diff
@@ -30,6 +30,6 @@ fn check(assist_id: &str, before: &str, after: &str) {
             )
         });
 
-    let actual = action.edit.apply(&before);
+    let actual = actions[0].edit.apply(&before);
```

Comment:
Currently doctests will check the first action only which is fine by now I think, since there are no assits with multiple actions yet in the codebase.